pymedphys/_vendor/patchlogging.py

```diff
@@ -27,6 +28,9 @@
 
 
 def apply_logging_patch():
+    if sys.version_info.major > 3 or sys.version_info.minor >= 8:
```

Comment:
This only works because Python 2.7 was end of line.
As a rule of thumb, I would take the following approach without having prior knowledge of where earlier versions stopped incrementing their minor version.

if sys.version_info.major > 3 or (sys.version_info.major == 3 and sys.version_info.minor >= 8):